server/routes/staticContent/staticContentController.ts

```diff
@@ -69,6 +69,16 @@ export default class StaticContentController {
       template: 'serviceProviderReferrals/sessions/amend',
       description: 'IC-1087 (session details: amend)',
     },
+    {
+      path: '/service-provider/session/feedback/confirmationNextStepESOR',
+      template: 'serviceProviderReferrals/sessions/feedback/confirmationNextStepESOR',
+      description: 'IC-1177 (confirmation page - next step ESOR)',
+    },
+    {
+      path: '/serviceProviderReferrals/notifBannertable',
```

Comment:
We don't usually use abbreviations unless there's a real need to. And, as above, can we be consistent with the naming used for the existing session progress table?

```suggestion
      path: '/service-provider/session-progress-all-sessions-completed',
      template: 'serviceProviderReferrals/sessionProgressAllSessionsCompleted',
      description: 'IC-1078 (session progress - all sessions completed, with notification banner)',
```